opennft/opennft.py

```diff
@@ -211,6 +212,14 @@ def __init__(self, *args, **kwargs):
         self.outputSamples = {}
         self.musterInfo = {}
 
+        if config.START_MATLAB_SCRIPT_USE:
+            if platform.system() == 'Linux':
+                os.system('sh linux_startmatlab.sh')
```

Comment:
Better way: to use `subprocess` module and `run` function instead of `os.system`
see: https://docs.python.org/3.6/library/subprocess.html#subprocess.run